src/editors/containers/ProblemEditor/data/MarkDownParser.js

```diff
@@ -38,28 +29,24 @@ function getHints(testMarkdown) {
     const hint = getHint(row);
     if (hint.length) {
       hints.push({
-        id: isEnvTest ? hints.length : Math.random(),
+        id: Number(d),
```

Comment:
what is `d` in this context?

Reply:
The idea here was to generate unique `id` for Hints and index looks like a better way to do that.

Reply:
`id: hint.length` would be a better approach I guess.